BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppAuthOperation.h

```diff
@@ -0,0 +1,74 @@
+//
+//  BOXAPIAppAuthToJSONOperation.h
```

Comment:
Remove this file, it isn't in the project and is redundant with BOXAPIAppUsersAuthOperation